test/options_test.cc

```diff
@@ -276,6 +277,141 @@ TEST_F(OptionsImplTest, RequestSource) {
   EXPECT_TRUE(util(*(options_from_proto.toCommandLineOptions()), *cmd));
 }
 
+// This test covers --RequestSourcePlugin, which can't be tested at the same time as --RequestSource
+// and some other options. This is the test for the inlineoptionslistplugin.
```

Comment:
nit:  If I understand correctly, this isn't a test of inlineoptionslistplugin.  It is a test of --RequestSourcePlugin that happens to use the inlineoptionslistplugin as its test requeest source plugin.

This is important because it would be weird to have a test of inlineoptionslistplugin in this file.

Reply:
That makes sense. Thanks dubious90. I see what you mean.